pkg/plugins/golang/v3/scaffolds/internal/templates/main.go

```diff
@@ -113,9 +113,11 @@ const (
 		os.Exit(1)
 	}
 `
-	webhookSetupCodeFragment = `if err = (&%s.%s{}).SetupWebhookWithManager(mgr); err != nil {
-		setupLog.Error(err, "unable to create webhook", "webhook", "%s")
-		os.Exit(1)
+	webhookSetupCodeFragment = `if os.Getenv("ENABLE_WEBHOOKS") != "false" {
+		if err = (&%s.%s{}).SetupWebhookWithManager(mgr); err != nil {
+			setupLog.Error(err, "unable to create webhook", "webhook", "%s")
+			os.Exit(1)
+		}
```

Comment:
This plugin (go/v3) is also deprecated. 
Could you please remove the change here?

Reply:
removed.